### PR TITLE
UI change for routes page

### PR DESCRIPTION
**Pull request by abrugal**
# Describe your changes
Make the text in the routes page a lot bigger.

Capture2

I also moved the buttons to the top because before if the list of routes got too long then the buttons would cover up the text.  Example of that issue in this second pic (ignore the colors lol)

Capture

# Issue ticket number and link
N/A

### CODE REVIEW

**Comment by AlexRutledge1** [comment]
Can you add a screenshot of what the final product looks like?

**Comment by AlexRutledge1** [comment]
Or is the first ss the final product?

**Comment by abrugal** [comment]
First pic is the final product. Second pic was just to show how the buttons were covering up the text when it was at the bottom

**Comment by AlexRutledge1** [comment]
On my device the text is very very big, by chance does your browser make text smaller by default?
Here's how it looks for me (27" desktop screen):
image

**Comment by abrugal** [comment]
Hmmm, I'm using nativebase's responsive values for the fontsize, which I though would automatically account for screen size. My browser's font size is medium, and my zoom is 100%. My display is 32". I'll see if I can fix this

**Comment by AlexRutledge1** [comment]
In case it helps: my browser settings are 16pt font size (never changed it so I assume that's default)

**Comment by abrugal** [comment]
Added some breakpoints, lmk if it looks good on your device

**Comment by AlexRutledge1** [comment]
Looks better, but let's scale down the `RouteRow` text one level so that the smallest is medium. It being the same size as the label seems weird to me.

File: src/components/RouteRow.tsx [hunk 1]
```diff
@@ -31,12 +31,22 @@ export const RouteRow = ({ route }: Props) => {
...
+            base: 'md',
+            lg: 'xl',
+            xl: '3xl'
+          }}>{data.name}</Text>
+          <Box justifyContent='center'>
+            <Text p='2' fontSize={{
+              base: 'md',
+              lg: 'xl',
+              xl: '3xl'
+            }}>{data.gradeDisplayString}</Text>
```
Comment (AlexRutledge1) [inline src/components/RouteRow.tsx]: Sorry I wasn't clear in the last comment, I think for each option the size should be one step down from the label size:
```ts
base: 'md',
lg: 'lg',
xl: '2xl'
```

Comment (abrugal) [reply src/components/RouteRow.tsx]: They were already 1 size smaller than the label's size. Added your change either way

**Comment by AlexRutledge1** [comment]
This looks good, but let's get @DanielW48's eyes on this since he has ownership over this page.

File: src/pages/Routes.tsx [hunk 2]
```diff
@@ -58,98 +58,96 @@ const Routes = () => {
...
+          </Box>
+          <Box w='1/2' alignItems='center'>
+            <Text bold  fontSize={{
+              base: 'lg',
+              lg: '2xl',
+              xl: '4xl'
+            }}>Archived Routes</Text>
             {isLoading ? <Text>Loading...</Text> :
               archivedRoutes.map((currRoute: Route) => (
                 <VStack key={currRoute.docRef?.id} width='100%'>
```
Comment (DanielW48) [inline src/pages/Routes.tsx:166]: I think this VStack should be a Scroll View come to think of it? or will it automatically allow us to scroll the whole page?

**Comment by DanielW48** [comment]
On my machine the text looks oversized now:
image

Was there some solution to having different CSS depending on the view?

**Comment by abrugal** [comment]
Is that oversized? I think that size looks good. Text gets smaller if screen gets smaller

**Comment by DanielW48** [comment]
> Is that oversized? I think that size looks good. Text gets smaller if screen gets smaller

I like it a little bit smaller if that is doable, otherwise looks nice

File: src/pages/Routes.tsx (hunk 2 above)
Comment (abrugal) [reply src/pages/Routes.tsx:166]: Sorry I originally didn't see this. It'll just let us scroll through the whole page. Scrollview's are specifically for react native

**Comment by DanielW48** [comment]
looks better, thanks man. hit that main merge and slam it i reckon

**Review by DanielW48 — APPROVED** [review]
LGTM

**Pull request merged**